packages/wagmi/src/utils/defaultWagmiReactConfig.ts

```diff
@@ -35,12 +29,13 @@ export function defaultWagmiConfig({
   metadata,
   enableInjected,
   enableCoinbase,
-  enableWalletConnect,
   enableEmail,
-  enableEIP6963,
-  ...wagmiConfig
+  enableWalletConnect,
+  enableEIP6963
 }: ConfigOptions): Config {
   const connectors: CreateConnectorFn[] = []
+  const transportsArr = chains.map(chain => [chain.id, http()])
```

Comment:
I'm wondering if we want to use wc transport at all here? 
We used to have it as a custom transport in wagmi1 but I couldn't find a way to provide it as a new option rather than the only one thus I left the `http` one, but maybe we want it for tracking or similar